prisma/schema.prisma

```diff
@@ -65,3 +56,10 @@ enum Status {
   INVITED
   DEACTIVATED
 }
+
+enum Subject {
```

Comment:
Is there a reason it's an enum now? thats what we had originally but then we were told that they wanted it as a model. also if it isnt an enum, we could later add functionality for adding subjects later on

Reply:
yeah Subject is a Model rn, just realised that the datamodels had been changed later on so they weren't updated in this branch will make the change